uint16 gamecode
uint16 ammoleft
uint16 timeleft
uint16 selfhealth
uint32 exteventdata
